packages/console/src/components/Button/index.tsx

```diff
@@ -6,18 +6,25 @@ import { useTranslation } from 'react-i18next';
 import * as styles from './index.module.scss';
 
 type Props = Omit<HTMLProps<HTMLButtonElement>, 'type' | 'size' | 'title'> & {
+  buttonType?: 'button' | 'submit' | 'reset';
```

Comment:
I think `buttonType` and `type` are confusing... consider changing it to `action` maybe?

Reply:
`buttonType` is the native `type` for button, can't figure out a better idea - any thoughts?

Reply:
Either change `buttonType` to `action`, or change `type` to `pattern`? I prefer changing `buttonType` to `action`

Reply:
"action" won't match the actual "type" property, which will leave this name mysterious

Reply:
`htmlType`

Reply:
@wangsijie yeah buddy @demonzoo ok with `htmlType`?

Reply:
Sounds good!